release/serve_tests/workloads/deployment_graph_wide_ensemble.py

```diff
@@ -0,0 +1,189 @@
+"""
+Test that focuses on wide fanout of deployment graph
+       -> Node_1
+      /          \
+INPUT --> Node_2  --> combine -> OUTPUT
+      \    ...   /
+       -> Node_10
+
+ 1) Intermediate blob size can be large / small
+ 2) Compute time each node can be long / short
+ 3) Init time can be long / short
+"""
+
+import time
+import asyncio
+import click
+
+from typing import Optional
+
+import ray
+from ray import serve
+from ray.experimental.dag import InputNode
+from ray.serve.drivers import DAGDriver
+from serve_test_cluster_utils import (
+    setup_local_single_node_cluster,
+    setup_anyscale_cluster,
+)
+from serve_test_utils import save_test_results
+from benchmark_utils import benchmark_throughput_tps, benchmark_latency_ms
+
+DEFAULT_FANOUT_DEGREE = 4
+
+DEFAULT_NUM_REQUESTS_PER_CLIENT = 20  # request sent for latency test
+DEFAULT_NUM_CLIENTS = 1  # Clients concurrently sending request to deployment
+
+DEFAULT_THROUGHPUT_TRIAL_DURATION_SECS = 10
+
+
+@serve.deployment
+class Node:
+    def __init__(self, id: int, init_delay_secs=0):
+        time.sleep(init_delay_secs)
```

Comment:
In ray dag each dag run re-creates the actor that runs init again, so when init cost is high it could lead to perf issue. For serve i wouldn't expect this to happen, but DAGNode's internal mechanism do create new DAGNode instances upon each execution.

So i put it here not as active flag but give us a tool to verify if init cost will play unintended bigger role in deployment graph